arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -516,5 +516,40 @@ public async Task DeleteVertexCollectionAsync_ShouldThrow_WhenVertexIsNotFound()
             Assert.Equal(HttpStatusCode.BadRequest, ex.ApiError.Code);
             Assert.Equal(1928, ex.ApiError.ErrorNum); // GRAPH_NOT_IN_ORPHAN_COLLECTION
         }
+
+        [Fact]
+        public async Task DeleteVertexCollectionAsync_ShouldDropCollection_WhenDropCollectionIsTrue()
+        {
+            // Create a new graph
+
+            string graphName = "DeleteVertexCollectionAsync_ShouldThrowNotFound_WhenCollectionDropIsTrue";
+
+            PostGraphResponse createResponse = await _client.PostGraphAsync(
+                new PostGraphBody()
+                {
+                    Name = graphName
+                });
+
+            // Add a vertex collection
+
+            string clxToDelete = "DeleteVertexCollectionAsync_ShouldDropCollection";
+
+            PostVertexCollectionResponse createVertexResponse = await _client.PostVertexCollectionAsync(
+                graphName,
+                new PostVertexCollectionBody()
+                {
+                    Collection = clxToDelete
+                });
+
+            var response = await _client.DeleteVertexCollectionAsync(graphName, clxToDelete, new DeleteVertexCollectionQuery
+            {
+                DropCollection = true
+            });
+
+            var collection = await _client.GetVertexCollections(graphName);
```

Comment:
I think we should use the collection client to test that the collection has actually been dropped? Because `GetVertexCollections` should obviously not return the vertex collection if it's been removed from the graph.

From what I understand, dropping the collection means deleting it. So we should check that this collection does exist anymore. Or am I mistaken? 🤔